### DISCUSSION TITLE
Chromebook

### DISCUSSION THREAD

**Discussion by T-Dawg138** [Prodigy-Hacking/ProdigyMathGameHacking#1182] [Questions] [answered]
How would we be able to do it on a Chromebook?

**Comment by donaldli2020** [comment]
Use a personal account

**Comment by donaldli2020** [comment]
***Then*** install the extension.

**Comment by ghost** [comment, marked as answer]
if school blocks the hacks, you cant do annything about it

**Reply by donaldli2020** [reply]
Or you can use a personal account.

**Reply by ghost** [reply]
no, you cant. school can block you from switching accounts. restrictions can be applied to the device and not the account.

**Reply by donaldli2020** [reply]
Some do, some don't.

**Reply by ghost** [reply]
i said can not always

**Reply by S0vereignity** [reply]
School computers kill you. PCs rule and help you. So do laptops
